src/LibraryWindow.cpp

```diff
@@ -64,18 +69,23 @@ LibraryWindow::loadFromLibrary(Library* _library)
   ui.imagesListView->setModel(imagesModel);
   ui.documentsListView->setModel(documentsModel);
   ui.dataListView->setModel(dataModel);
+  
+  ui.tagsListView->setModel(tagsModel);
+  ui.currentTagsListView->setModel(currentTagsModel);
+
+  /* load tags */
+  std::cout << "Calling display tags for library: " << library->name() << std::endl;
+  loadTags();
+  std::cout << "Called display tags for library: " << library->name() << std::endl;
 }
 
 
-void
```

Comment:
If you're going to update style on methods, you will need to be prepared to update all of them for consistency.